packages/emulation/libretro-2048/package.mk

```diff
@@ -4,7 +4,7 @@
 PKG_NAME="libretro-2048"
 PKG_VERSION="331c1de588ed8f8c370dcbc488e5434a3c09f0f2"
 PKG_SHA256="9b30278e61b8dfe067d8f7cc9cb0d467e16a3d2de995e950b712570d7e4aa195"
-PKG_LICENSE="GPLv3"
+PKG_LICENSE="Public domain"
```

Comment:
Also should be fixed upstream: https://github.com/libretro/Lakka-LibreELEC/blob/Lakka-v5.x/packages/lakka/libretro_cores/2048/package.mk

Reply:
I'll PR after this is merged